src/validators/function.rs

```diff
@@ -442,13 +442,13 @@ macro_rules! py_err_string {
 /// Only `ValueError` (including `PydanticCustomError` and `ValidationError`) and `AssertionError` are considered
 /// as validation errors, `TypeError` is now considered as a runtime error to catch errors in function signatures
 pub fn convert_err<'a>(py: Python<'a>, err: PyErr, input: &'a impl Input<'a>) -> ValError<'a> {
-    if err.is_instance_of::<PyValueError>(py) {
+    if let Ok(validation_error) = err.value(py).extract::<ValidationError>() {
```

Comment:
pls can we revert this - failed `extract` can be quite slow.

Reply:
```suggestion
    if let Ok(validation_error) = err.is_instance_of::<ValidationError>(py) {
```